dev-tools/reconfigurator-cli/tests/output/cmds-target-release-stdout

```diff
@@ -7991,12 +7968,38 @@ to:   blueprint 12f19448-6a10-4b4b-ae19-a8c94a566097
 
 
 internal DNS:
-  DNS zone: "control-plane.oxide.internal" (unchanged)
-    unchanged names: 54 (records: 71)
+* DNS zone: "control-plane.oxide.internal": 
+-   name: 0c71b3b2-6ceb-4e8f-b020-b08675e83038.host          (records: 1)
+-       AAAA fd00:1122:3344:101::22
+-   name: 3eeb8d49-eb1a-43f8-bb64-c2338421c2c6.host          (records: 1)
+-       AAAA fd00:1122:3344:103::22
+-   name: 466a9f29-62bf-4e63-924a-b9efdb86afec.host          (records: 1)
+-       AAAA fd00:1122:3344:102::22
++   name: 90dbd6f3-9bcb-4a62-ad85-d61f5b3a36ad.host          (records: 1)
++       AAAA fd00:1122:3344:103::2b
++   name: 9ae90740-7fdb-4073-ae43-048f2fca3d69.host          (records: 1)
++       AAAA fd00:1122:3344:102::2c
+*   name: _nexus._tcp                                        (records: 3 -> 3)
+-       SRV  port 12221 0c71b3b2-6ceb-4e8f-b020-b08675e83038.host.control-plane.oxide.internal
+-       SRV  port 12221 3eeb8d49-eb1a-43f8-bb64-c2338421c2c6.host.control-plane.oxide.internal
+-       SRV  port 12221 466a9f29-62bf-4e63-924a-b9efdb86afec.host.control-plane.oxide.internal
++       SRV  port 12221 90dbd6f3-9bcb-4a62-ad85-d61f5b3a36ad.host.control-plane.oxide.internal
++       SRV  port 12221 9ae90740-7fdb-4073-ae43-048f2fca3d69.host.control-plane.oxide.internal
++       SRV  port 12221 d516d61b-fd96-46ad-a743-78eec814ee90.host.control-plane.oxide.internal
++   name: d516d61b-fd96-46ad-a743-78eec814ee90.host          (records: 1)
++       AAAA fd00:1122:3344:101::2b
+    unchanged names: 47 (records: 59)
 
 external DNS:
-  DNS zone: "oxide.example" (unchanged)
-    unchanged names: 5 (records: 12)
+* DNS zone: "oxide.example": 
+*   name: example-silo.sys                                   (records: 3 -> 3)
+-       A    192.0.2.2
+-       A    192.0.2.3
+-       A    192.0.2.4
++       A    192.0.2.7
++       A    192.0.2.6
++       A    192.0.2.5
+    unchanged names: 4 (records: 6)
```

Comment:
Similarly, this change reflects the bug fixed by this PR.  Before, we had already updated DNS (wrongly) at this point in the process.  This is the correct time to do it (we've just done the handoff).